numpy/core/src/multiarray/descriptor.c

```diff
@@ -1724,12 +1724,12 @@ _convert_from_str(PyObject *obj, int align)
 
         /* Check for a deprecated Numeric-style typecode */
```

Comment:
This whole block no evaluates to nothing (or apparently a compiler error in MSVC).  You can just delete it, since the whole for loop will never run (would have length 0).

Reply:
The whole thing including the loop and the variables should be deleted, correct?

Reply:
yes